tests/unit/preprocessor/RandomFourierGaussPreproc_unittest.cc

```diff
@@ -0,0 +1,69 @@
+/*
+ * This software is distributed under BSD 3-clause license (see LICENSE file).
+ *
+ * Author: Nanubala Gnana Sai
+ */
+
+#include <gtest/gtest.h>
+#include <shogun/features/DenseFeatures.h>
+#include <shogun/kernel/GaussianKernel.h>
+#include <shogun/mathematics/NormalDistribution.h>
+#include <shogun/mathematics/linalg/LinalgNamespace.h>
+#include <shogun/preprocessor/DensePreprocessor.h>
+#include <shogun/preprocessor/RandomFourierGaussPreproc.h>
+
+using namespace shogun;
+
+class RFGPTest : public ::testing::Test
```

Comment:
it is a good idea to make such a fixture class.... IF there are multiple tests sharing the same code, which is not the case here. Did you plan to add more tests? If so which ones? I tend to think that a single test is enough ... ?

Reply:
you could test: apply to feature matrix, same for vector, and then the apply method of the transformer (unless that is tested elsewhere)

Reply:
what about the above q? There is still only a single test

Reply:
@karlnapf , forgive the delay, I wanted to make sure the first test is working well before I added another. Also, ```apply_to_feature_matrix``` is hidden so we can use only ```apply_to_vectors``` and ```transform``` both of which are tested and working!